### PR TITLE
Try redesigning for a better homepage

### PR DESCRIPTION
**Pull request by celestehorgan**
Signed-off-by: Celeste Horgan <celeste@cncf.io>

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -9,9 +9,9 @@ style = "paraiso-dark"
 
 [params]
 font_awesome_version = "5.12.0"
-description = "A project for inclusive language and terminology in open source and technology"
+description = "Helping projects and companies make consistent, responsible choices to remove harmful language"
```
Comment (edwarnicke) [inline config.toml:12]: Can this be phrased in the positive?

File: config.toml [hunk 2]
```diff
@@ -9,9 +9,9 @@ style = "paraiso-dark"
...
 favicon = "favicon.png"
-repositoryUrl = "https://github.com/cncf/hugo-netlify-starter"
+repositoryUrl = "https://github.com/inclusivenaming"
```
Comment (edwarnicke) [inline config.toml:14]: Good catch :)

File: content/about.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: "About the Inclusive Naming Initiative"
+---
+
+The Inclusive Naming Initiative’s mission is to help companies and projects remove all harmful and unclear language of any kind and replace it with an agreed-upon set of neutral terms. The initiative’s goal is to define processes and tools to remove harmful language from projects. This includes creating a comprehensive list of terms with replacements, language evaluation frameworks and templates, and infrastructure to aid the transition. RedHat, IBM, LF Networking, and the Kubernetes Naming WG are all already on board.
```
Comment (edwarnicke) [inline content/about.md:5]: Can this be phrased in the positive?  'remove' and 'harmful' carry negative energy with them.  Sometimes that's needful and appropriate, but I often find there is great power in rephrasing in the positive.

File: config.toml (hunk 1 above)
Comment (celestehorgan) [reply config.toml:12]: @xmulligan any thoughts?

I don't think this is a blocker for merge, but it'd be good to revise.

File: content/about.md (hunk 3 above)
Comment (celestehorgan) [reply content/about.md:5]: @xmulligan? 

Again, i don't think this is a blocker for merging this PR. We can iterate.

Comment (edwarnicke) [reply content/about.md:5]: Absolutely not a blocker.  Please do merge.

File: config.toml (hunk 1 above)
Comment (edwarnicke) [reply config.toml:12]: Absolutely not a blocker.  Please do merge.

**Comment by edwarnicke** [comment]
Never let the nit picky be the enemy of the good :)

**Pull request merged**